feature/system/tests/system_base_test/g_protocol_test.go

```diff
@@ -58,7 +59,22 @@ func TestGNMIClient(t *testing.T) {
 	dut := ondatra.DUT(t, "dut")
 	conn := dialConn(t, dut, introspect.GNMI, 9339)
 	c := gpb.NewGNMIClient(conn)
-	if _, err := c.Get(context.Background(), &gpb.GetRequest{Encoding: gpb.Encoding_JSON_IETF, Path: []*gpb.Path{{Elem: []*gpb.PathElem{}}}}); err != nil {
+
+	var req *gpb.GetRequest
```

Comment:
this variable is unused if this deviation is not met can we move it to inside the if block

Reply:
`req`? It is actually used in the else block as well.